website/docs/d/is_lb_pool.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : ibm_is_lb_pool"
+description: |-
+  Get information about LoadBalancerPool
+---
+
+# ibm_is_lb_pool
+
+Provides a read-only data source for LoadBalancerPool. For more information, about load balancer pool, see [working with pool](https://cloud.ibm.com/docs/vpc?topic=vpc-nlb-pools).
+
+## Example Usage
+
+```terraform
+data "ibm_is_lb_pool" "example" {
+	identifier = ibm_is_lb.example.id
+	lb = element(split("/",ibm_is_lb_pool.example.id),1)
```

Comment:
these examples are wrong 
ibm_is_lb.example.id gives the id of lb not the identifier (pool id)

Reply:
they were referenced in a reverse order, changed them..
Thank you for noticing